lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -4284,7 +4286,8 @@ void LLVMToSPIRVBase::transGlobalAnnotation(GlobalVariable *V) {
         cast<GlobalVariable>(CS->getOperand(1)->stripPointerCasts());
 
     StringRef AnnotationString;
-    getConstantStringInfo(GV, AnnotationString);
+    if (!getConstantStringInfo(GV, AnnotationString))
+      assert(0 && "Annotation string missing");
```

Comment:
```suggestion
    if (!getConstantStringInfo(GV, AnnotationString)) {
      assert(!"Annotation string missing");
      return;
    }
```
Or an error. Though I believe it's no fatal enough to abort translation, at some point we should have warnings added.